# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/nvi/ncurses.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./PORT/linux/Makefile	1995-03-18 21:02:03.000000000 +0100
+++ ./PORT/linux/Makefile	2002-09-26 03:41:18.000000000 +0200
@@ -42,19 +42,19 @@
 # To use the curses library included in the distribution, uncomment these
 # lines.
 #
-CDEF=	-DDef_term=xDef_term -Dendwin=xendwin -Dttytype=xttytype
-CURSES=-Icurses
-OBJ12=	addbytes.o addch.o addnstr.o box.o clear.o clrtobot.o clrtoeol.o \
-	cr_put.o ctrace.o cur_hash.o curses.o delch.o deleteln.o delwin.o \
-	erase.o fullname.o getch.o getstr.o id_subwins.o idlok.o initscr.o \
-	insch.o insertln.o longname.o move.o mvwin.o newwin.o overlay.o \
-	overwrite.o putchar.o refresh.o scroll.o setterm.o standout.o \
-	toucholap.o touchwin.o tscroll.o tstp.o tty.o unctrl.o
+#CDEF=	-DDef_term=xDef_term -Dendwin=xendwin -Dttytype=xttytype
+#CURSES=-Icurses
+#OBJ12=	addbytes.o addch.o addnstr.o box.o clear.o clrtobot.o clrtoeol.o \
+#	cr_put.o ctrace.o cur_hash.o curses.o delch.o deleteln.o delwin.o \
+#	erase.o fullname.o getch.o getstr.o id_subwins.o idlok.o initscr.o \
+#	insch.o insertln.o longname.o move.o mvwin.o newwin.o overlay.o \
+#	overwrite.o putchar.o refresh.o scroll.o setterm.o standout.o \
+#	toucholap.o touchwin.o tscroll.o tstp.o tty.o unctrl.o
 
 # To use the system's curses library, or to include additional
 # routines from the system's curses library, uncomment this line.
 #
-#CULIB=	-lcurses
+CULIB=	-lncurses
 
 # To use a System V curses interface with the system's curses library,
 # uncomment this line.  Using a System V curses library is dangerous.
@@ -93,7 +93,7 @@
 
 # Some vendors use "-ltermcap", not "-ltermlib".
 bin/nvi: options.h excmd.h ${ALL}
-	${CC} ${OORG} -o $@ ${ALL} ${CULIB} -ltermcap
+	${CC} ${OORG} -o $@ ${ALL} ${CULIB}
 	(cd bin; ln -sf nvi nex)
 
 options.h:
